packages/connect/e2e/__fixtures__/solanaGetAddress.ts

```diff
@@ -0,0 +1,47 @@
+const legacyResults = [
+    {
+        // Solana not supported below this version
+        // TODO solana (vl/connect): set proper version
+        rules: ['<2.7.0', '1'],
+        success: false,
+    },
+];
+
+export default {
```

Comment:
do you have tests fixtures in firmware repo? if yes, preferred way would be to import them from submodules folder probably to avoid duplication

Reply:
Yes there are tests in the FW repo and yes they are also the same. I'm not sure what the steps would be to be able to import the tests here. The FW PR has still not been merged and so the `trezor-common` repo has not been updated as well - I don't know what the process is to update it.

To be honest I'd prefer if we didn't block merging this PR on this detail, we can refactor the tests once `trezor-common` has been updated.

Probably only the `solanaGetAddress` and `solanaGetPublicKey` tests could be reused from FW - we haven't really gotten feedback yet on how to handle the sign tx test on FW but they most likely won't be usable in Connect.

And I just noticed that the `solanaSignTransaction` tests are missing the `path` parameter - I will fix this.

Reply:
> And I just noticed that the solanaSignTransaction tests are missing the path parameter - I will fix this.

Fixed in https://github.com/trezor/trezor-suite/pull/9958/commits/2f51430f6a270797021b5d7c271b49fccfa3d598.

Reply:
ok